hoot-core/src/main/cpp/hoot/core/cmd/DiffCmd.cpp

```diff
@@ -91,6 +91,8 @@ class DiffCmd : public BaseCommand
 
     OsmMapPtr map1(new OsmMap());
     IoUtils::loadMap(map1, args[0], true, Status::Unknown1);
+    //  Some maps that don't have IDs cooked in will fail comparison if the IDs aren't reset
+    OsmMap::resetCounters();
```

Comment:
@bwitham using `hoot diff` was a good idea.  I only had to change this to reset the counters because GeoJSON doesn't contain IDs so resetting them allowed the maps to match.  Thanks for the idea.

Reply:
cool